composer/datasets/ade20k.py

```diff
@@ -112,6 +140,87 @@ def build_ade20k_dataloader(
     )
 
 
+def build_streaming_ade20k_dataloader(
+    batch_size: int,
+    remote: str,
+    *,
+    version: int = 2,
+    local: str = '/tmp/mds-cache/mds-ade20k/',
```

Comment:
Does `local` variable path always exist in a machine? I am wondering if user doesn't provide it and it fails with `FileNotFound` Exception!!

Reply:
I was under the impression the streaming dataset created the local folder, is this not the case?

Reply:
I'm pretty sure it does.

Reply:
Yes it does [here](https://github.com/mosaicml/streaming/blob/main/streaming/base/download.py#L123)